metaflow/cli.py

```diff
@@ -718,12 +717,12 @@ def version(obj):
               show_default=True,
               help='Measure code coverage using coverage.py.')
 @click.option('--event-logger',
-              default='nullSidecarLogger',
+              default=DEFAULT_EVENTLOGGER,
```

Comment:
Should this be `DEFAULT_EVENT_LOGGER` considering there is a hyphen in the name?
Similar comment for `DEFAULT_PACKAGE_SUFFIXES`? I'm assuming the consistency could help but I feel weakly about this comment, so let me know if you disagree.